### ISSUE TITLE
mvn test resulting in mongodb exception

### ISSUE THREAD

**Issue by spw1016** [Chicago/opengrid-svc-template#11]
When I try to run `mvn test`, i get the error:

`INFO: Exception in monitor thread while connecting to server localhost:27017
com.mongodb.MongoSecurityException: Exception authenticating MongoCredential{mechanism=null, userName='build', source='testdb', password=<hidden>, mechanismProperties={}}`

I imagine I am skipping some step here with Mongodb, but it's a bit hard to follow as the documentation appears to be unfinished for that section. Would someone be willing to help me out? I'd be more than happy to fill in the documentation or clarify it for others who might want to help develop opengrid.

**Comment by rladines** [comment]
I will take a look at this, will try to recreate.
